tests/test_unit_parse_yokogawa_metadata.py

```diff
@@ -27,7 +27,7 @@
 mlf_path_1 = f"{path}MeasurementData_SingleWell2Sites_MultiZ.mlf"
 mrf_path_1 = f"{path}MeasurementDetail.mrf"
 expected_files_1 = 4
-expected_shape_1 = (2, 13)
+expected_shape_1 = (2, 11)
```

Comment:
Can you remind me why is that changing?

Reply:
I've moved the remove_FOV_overlaps function from the parse_yokogawa_metadata into the parsing tasks themselves. It's only applied for some versions of parsing the metadata (when things are read directly from the mrf_mlf file), but it's useful to be able to parse the metadata without changing the overlaps (e.g. as part of workflows to manually adapt the metadata).
The only test it made fail was the unit_test for the parsing, because the dimensions of the output changed (now, it doesn't contain the original positions anymore)
(see also comment above)